### PR TITLE
Single player

### PR DESCRIPTION
**Pull request by TastingComb**
1. made a Single player mode possible (changed minimum number of JSpinner to 1
2. implemented a JLabel to show when you don't get a six to move out of base (with jButton to move on to next player)
3. implemented an message for the user that shows the bot movements, so user can see what's happening in the BackEnd

### CODE REVIEW

File: BackEnd.java [hunk 1]
```diff
@@ -117,8 +119,7 @@ private void playerMoveOnField() {
         }
 
 	if (beatsPossible) {
-	    // Only figures able to beat another figure should be
```
Comment (guemax) [inline BackEnd.java:120]: I added a hard line break in this comment to keep the line width under 72 characters.

Sticking with that number might seem odd (it is from a time where programmers used punch cards), but it really simplifies reading code when I have two windows side by side as the code fits perfectly on that limited amount of space.

File: BackEnd.java [hunk 2]
```diff
@@ -117,8 +119,7 @@ private void playerMoveOnField() {
...
-	    // moved now.
```
Comment (guemax) [inline BackEnd.java:121]: I added a hard line break in this comment to keep the line width under 72 characters.

Sticking with that number might seem odd (it is from a time where programmers used punch cards), but it really simplifies reading code when I have two windows side by side as the code fits perfectly on that limited amount of space.

**Review by MeiNic — CHANGES_REQUESTED** [review]
There is a lot to do in the `backEnd.java` file:
1. Please remove/add **ALL** unnecessary indentations, bescause this code is nearly unreadable (some of the changes are only unnecesary added indentations e.g. line 321 till 324)
2. Remove **ALL** empty lines unnecessary added
3. Remove the unused variable: `movedFieldsNew` (line 170)
4. Remove variable `movedFields` (values is always the same as the variable `randomNumber`)
5. Add the linebreak between line 120 and 121 (see the comment from [guemax](https://github.com/guemax))

There is also following bug:
After one or more bots made their move, the result for the normal user isn't displayed anymore.
Could you fix this as well?

**Comment by MeiNic** [comment]
This PR is closed, until the structure-updates from @guemax are on the master-branch.
Then the need changes for the `SinglePlayer`-Mode can may be implemented in another branch, because otherwise there into big difference and it gets complicated to merge this PR.

**Pull request closed without merging**